src/store/auth/actions.tsx

```diff
@@ -0,0 +1,14 @@
+import { AuthState, LOGIN, LOGOUT } from './types';
+
+export const login = (userInfo: AuthState) => {
```

Comment:
What if this function is called with `{loginState: 'ANONYMOUS'}`? Does it make sense?
I suggest changing the argument to `{uid, username}`.